pkg/registrymirror/store_test.go

```diff
@@ -336,7 +336,7 @@ func TestPullSources(t *testing.T) {
 		assert.NoError(t, err)
 
 		err = s.UpsertImageDigestMirrorSet(idmsA)
-		assert.Error(t, err)
+		assert.NoError(t, err)
```

Comment:
@dcaravel Could you review this part. After updating runtime utiles this is now supported. See:
- https://github.com/openshift/runtime-utils/pull/18

Reply:
The purpose of this and the following test are to validate behavior when an 'error' occurs, would prefer we force an error.  Passing a bad path triggers an error - considering using this instead:

```go
	t.Run("error when CRs updated to bad state (sync)", func(t *testing.T) {
		badPath := ""

		s := NewFileStore(WithConfigPath(badPath), WithDelay(0))

		err := s.UpsertImageContentSourcePolicy(icspA)
		assert.Error(t, err)
	})

	t.Run("no error when CRs are updated to bad state (async)", func(t *testing.T) {
		// the config update will fail and the failure logged, however because the write
		// is async no error is returned.
		badPath := ""

		s := NewFileStore(WithConfigPath(badPath), WithDelay(30*time.Millisecond))

		err := s.UpsertImageContentSourcePolicy(icspA)
		time.Sleep(40 * time.Millisecond)
		assert.NoError(t, err)
	})
```